### ISSUE TITLE
Documentation of distributions: include PDF

### ISSUE THREAD

**Issue by dhardy** [boxtown/statrs#91]
@boxtown should we make it a policy to document the PDF of all distributions, where possible?

I think the [GSL documentation](https://www.gnu.org/software/gsl/doc/html/randist.html#) is a good model for what documentation of distributions should be.

Specifically, as noted in https://github.com/rust-random/rand/pull/621, your implementation of the Geometric distribution does not specify which variant is implemented, aside from the adjective "shifted" in the constructor. I recommend something more like [this](https://www.gnu.org/software/gsl/doc/html/randist.html#the-geometric-distribution).

**Comment by boxtown** [comment]
Yeah I agree that if there's qualifying information that is missing we should definitely add it where we can. I do want to be careful about toe-ing the line between useful and redudant information though
